src/banana/benchmark/runner.py

```diff
@@ -48,10 +54,33 @@ def get_pdf(pdf_name):
             if len(res.stdout) == 0:
                 raise ValueError("lhapdf could not install the set!")
             print(f"{pdf_name} installed.")
-        pdf = lhapdf.mkPDF(pdf_name, 0)
+        if full_set:
+            pdf = lhapdf.mkPDFs(pdf_name)
+        else:
+            pdf = lhapdf.mkPDF(pdf_name, 0)
     return pdf
 
 
+def get_pdf_name(pdf):
```

Comment:
As usual: in a more pythonic way you could name the function `pdf_name`, since it contains already all the information (`get_` for getters is a common pattern, but in other languages, and python explicitly suggest to avoid).

It's just a minor thing...